### ISSUE TITLE
could you tell me the file format for flag --group_by?

### ISSUE THREAD

**Issue by blairzhang126** [vatlab/VariantTools#36]
Sorry to ask about this, there are too much information on the website and I couldn't find the file format for --group_by. Thank you very much!

**Comment by BoPeng** [comment]
You meant http://varianttools.sourceforge.net/Vtools/Associate and examples such as http://varianttools.sourceforge.net/Association/Collapsing ?

**Comment by blairzhang126** [comment]
Yes,correct. I meant for association analysis, but I'm a little confused by how "name2" looks like. I'm trying to create a file that has "group" information (I guess it includes gene and variant information). I'm doing rare variant analysis so that I need to provide a group information instead of single variant analysis. Thanks so much for your help Bo.

**Comment by BoPeng** [comment]
`name2` is the common gene name in refGene (in contrast to ID), you can check the meaning of these fields by command `vtools use ...`, `vtools show fields`.

**Comment by blairzhang126** [comment]
May I ask what's the format for "name2" file?

**Comment by BoPeng** [comment]
http://varianttools.sourceforge.net/Annotation/RefGene  if you are referring to the `name2` field of refGene

**Comment by blairzhang126** [comment]
I realized my problem. I thought the file after "--group_by" is the same or similar format as the SetID as in "SKAT" R package. Now I understand that I should load the annotation database and "name2" is a field name. Thank you very much!!! I got confused by "name2", I thought you guys have created some specific file called "name2". Anyway, sorry to bother you for such a long time and thank you very much!!!!
